lib/features/artifacts.js

```diff
@@ -142,7 +153,9 @@ export default class Artifacts {
       var res = yield waitForEvent(putReq, 'response');
```

Comment:
As far as co() it will be gone soon.  We need to rewrite the artifact handler and I was doing this in stages.